doc/user/bgp.rst

```diff
@@ -2934,6 +2941,21 @@ by route reflectors to avoid looping.
 .. index:: bgp cluster-id A.B.C.D
 .. clicmd:: bgp cluster-id A.B.C.D
 
+To set and unset the BGP daemon ``-n`` / ``--no_kernel`` options during runtime
```

Comment:
This paragraph shouldn't go under `bgp cluster-id`, I guess.

Reply:
I will put it after the command definition.